src/sagemaker/lineage/query.py

```diff
@@ -201,194 +201,101 @@ def _artifact_to_lineage_object(self):
-                                "border-opacity": "0",
-                                "border-color": "#232f3e",
-                                "font-family": "verdana",
-                            },
-                        },
-                        {
-                            "selector": "edge",
-                            "style": {
-                                "label": "data(label)",
-                                "color": "gray",
-                                "text-halign": "left",
-                                "text-margin-y": "2.5",
-                                "font-size": "3",
-                                "width": "1",
-                                "curve-style": "bezier",
-                                "control-point-step-size": "15",
-                                "target-arrow-color": "gray",
-                                "target-arrow-shape": "triangle",
-                                "line-color": "gray",
-                                "arrow-scale": "0.5",
-                                "font-family": "verdana",
-                            },
-                        },
-                        {"selector": ".select", "style": {"border-opacity": "0.7"}},
-                    ]
-                    + [self._create_entity_selector(k, v) for k, v in self.graph_styles.items()],
-                    responsive=True,
-                ),
-                self.html.Div(
-                    style={
-                        "width": "0.5%",
-                        "display": "inline-block",
-                        "font-size": "1vw",
-                        "font-family": "verdana",
-                        "vertical-align": "top",
-                    },
-                ),
-                # legend section
-                self.html.Div(
-                    [self._create_legend_component(v) for k, v in self.graph_styles.items()],
-                    style={
-                        "display": "inline-block",
-                        "font-size": "1vw",
-                        "font-family": "verdana",
-                        "vertical-align": "top",
-                    },
-                ),
-            ]
-        )
-
-        @app.callback(
-            self.Output("cytoscape-graph", "elements"),
-            self.Input("cytoscape-graph", "tapNodeData"),
-            self.Input("cytoscape-graph", "elements"),
-        )
-        def selectNode(tapData, elements):
-            for n in elements:
-                if tapData is not None and n["data"]["id"] == tapData["id"]:
-                    # if is tapped node, add "select" class to node
-                    n["classes"] += " select"
-                elif "classes" in n:
-                    # remove "select" class in "classes" if node not selected
-                    n["classes"] = n["classes"].replace("select", "")
+        return pyvis, Network, Options
 
-            return elements
+    def _get_options(self):
```

Comment:
Could probably be an instance variable rather than a get method (like described [here](https://google.github.io/styleguide/pyguide.html#315-getters-and-setters))

Reply:
changed to `self._options` declared in `__init__`